saml_idp.gemspec

```diff
@@ -41,6 +41,7 @@ defaults in a Production environment. Post any issues you to github.
   s.add_dependency('builder')
   s.add_dependency('httparty')
   s.add_dependency('nokogiri')
+  s.add_dependency('xmlenc', '>= 0.6.4')
```

Comment:
Yep that's what I'm picturing as well. We can add it to the install message I believe in the gemspec and error when people try to encrypt without the library also installed